package.json

```
@@ -60,8 +60,7 @@
     "react-dom": "^18",
     "react-redux": "^9.1.0",
     "swr": "^2.2.5",
-    "tapwrite": "^1.1.51",
-    "tippytappy": "^1.0.4",
+    "tapwrite": "^1.1.56",
```

We're using `^` for acceptable version check - this will cause major issues

If you push this to prod, then publish version 1.1.57 and 1.1.58 for a feature branch, the `main` branch will take 1.1.58 instead of 1.1.56 on the next re-deploy.

What `"tapwrite": "^1.1.56"` does is say I want a package version starting from 1.1.56 and lesser that 2.0.0. This is not what we want - we want a fixed version for 1.1.56.

```suggestion
    "tapwrite": "1.1.56",
```

---

https://docs.npmjs.com/about-semantic-versioning